syntax/import.go

```diff
@@ -81,32 +82,28 @@ func importModuleFile(ctx context.Context, importPath string) (rel.Expr, error)
 	if isRunningBundle(ctx) {
 		return fileValue(ctx, path.Join(ModuleDir, importPath))
 	}
-
-	wd, err := os.Getwd()
-	if err != nil {
-		return nil, err
+	repo, resource, ver, _ := gop.ProcessRequest(importPath)
+	if path.Ext(resource) == ""{
+		resource += ".arrai"
 	}
-	if err := mod.Config(mod.GoModulesMode,
-		mod.GoModulesOptions{ModName: filepath.Base(wd)}, mod.GitHubOptions{}); err != nil {
-		return nil, err
+	if ver == ""{
+		ver = "HEAD"
 	}
-
-	path, ver := mod.ExtractVersion(importPath)
-	if ver != "" {
-		return nil, errors.New("per-importing versioning is not allowed")
+	if repo != ""{
+		repo += "/"
 	}
-
-	m, err := mod.Retrieve(path, ver)
-	if err != nil {
+	r := mod.New(ctxfs.SourceFsFrom(ctx))
+	bytes, _, err := r.Retrieve(path.Join(repo,resource)+"@"+ver)
+	if err != nil{
 		return nil, err
 	}
-
-	relImportPath := strings.TrimPrefix(importPath, m.Name)
-	if err := bundleModule(ctx, relImportPath, m); err != nil {
```

Comment:
I don't know how the rest of this works, but this looks fishy. Does bundling still work?